.travis.yml

```diff
@@ -1,130 +1,121 @@
+# Runs Test scripts; Linting; Deployment on Travis-CI
 language: python
 python:
   # We don't actually use the Travis Python, but this keeps it organized.
   #- "2.7"    # not tested anymore starting from 0.9.19
   - "3.8"
+  - "3.9"
+  - "3.10"
```

Comment:
The more Python versions the faster we burn out Credits. I'd keep 1 Vaex-version only; and 1 non-Vaex version (after #656)  @minouHub  

So for the time being , let's test on Python 3.10  and drop 3.8; 3.9 tests ?

In the future we might want to test multiple versions only for merges to Master branch , but I don't know how to add that easily in the Travis matrix.
At some point we'll also switch to GitHub Actions; so let's do this multi-python testing at that time (GSOC project?)

Reply:
On the other hand ... if we had the tests ready for python 3.11 and 3.12, we would have caught the problem with vaex earlier. I don't know if the Travis credit are really a problem, we can always ask for a renewal right? On top of that, we now have the "fast" tests stopping the build and saving a lot of credits. I'd prefer to keep several python versions tested. Let's say three at least, distributed the way we prefer.

Reply:
On that: 

- [ ] let's drop the tests on Windows for the moment
- [ ] I'll look for a way to implement more tests but only on the master branch